packages/server/type-augmentations/express.d.ts

```diff
@@ -0,0 +1,10 @@
+import { Request } from 'express'
+import { AuthContext } from '@/modules/shared/authz'
+
+declare module 'express' {
+  interface RequestWithAuthContext extends Request {
```

Comment:
Just extend `Request` directly, no need to introduce a new interface. This will ensure that whenever you use the main express Request type, it has the property on it.

```ts
declare module 'express' {
  interface Request {
    context: AuthContext
  }
}
```